dashboard/src/containers/AppNewContainer/AppNewContainer.tsx

```diff
@@ -30,7 +30,7 @@ function mapStateToProps(
     chartNamespace: params.global === "global" ? config.kubeappsNamespace : params.namespace,
     cluster: params.cluster,
     chartVersion: params.version,
-    error: apps.error,
+    error: apps.error || charts.selected.error,
```

Comment:
when rendering the deployment form we also need to render the error when fetching the chart if exists.